xla/stream_executor/rocm/rocm_blas.cc

```diff
@@ -366,15 +366,18 @@ absl::Status ROCMBlas::DoBlasInternalImpl(FuncT rocblas_func, Stream *stream,
                  << ToString(ret);
     }
   }
-#if TF_ROCM_VERSION >= 60000
+#if 0
+// pemeliya: the feature is disabled since rocblas does not perform well under
+// graph capture. rocblas_set_workspace seems to use blocking memory functions
+// like hipFree/hipMalloc which result in HIP_ERROR_StreamCaptureUnsupported 
   {
     auto *workspace = GetWorkspace();
     auto *wptr = workspace != nullptr ? workspace->opaque() : nullptr;
     size_t wsize = workspace != nullptr ? workspace->size() : 0;
-    ret = wrap::rocblas_set_workspace(blas_, wptr, wsize);
+    ret = wrap::rocblas_set_workspace(blas_, wptr, wsize); 
     if (err_on_failure && ret != rocblas_status_success) {
-      LOG(ERROR) << "failed to set workspace before " << FuncT::kName << ": "
-                 << ToString(ret);
+      LOG(ERROR) << "failed to set workspace before " << FuncT::kName
+                 << ": " << ToString(ret);
```

Comment:
Hey @pemeliya, can you just delete the code? We can always bring it back when needed.

Feel free to leave a comment instead.

Reply:
sorry, for the late reply.. Yes I have deleted this code on the branch but looks like it was too late..